web/config.py

```diff
@@ -22,6 +22,14 @@
 if sys.path[0] != root:
     sys.path.insert(0, root)
 
+# The number of connections to keep open inside the connection pool.
+# A SQLITE_CONNECTION_POOL_SIZE setting of 0 indicates no limit.
+SQLITE_CONNECTION_POOL_SIZE = 7
+# The number of connections to allow in connection pool "overflow".
+# These connections that can be opened above and beyond the pool_size setting
+SQLITE_CONNECTION_MAX_OVERFLOW = 1000
```

Comment:
The default connection size of a PG server is 100. This should be at least lesser than that.
This setting will be used by external PostgreSQL dbs also. So prefixing `SQLITE_` is not appropriate. I would suggest using - `CONFIG_DATABASE_` of `CONFIG_DATABASE_URI`.